.github/workflows/index-prep-caller.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: Run scenario preparation
+
+# This workflow is based on, but diverges from the example here:
+# https://docs.docker.com/build/ci/github-actions/multi-platform/#distribute-build-across-multiple-runners
+
+on:
+  pull_request:
+  push:
+    branches: [main]
+  schedule:
+    - cron: '0 0 * * 1,2,3,4,5'
+  workflow_dispatch:
+
+jobs:
+  run-index-prep:
+    strategy:
+      # fail if any build fails
+      fail-fast: false
+      # build amd64 and arm64 in parallel
+      matrix:
+        include:
+          - config_active: "2022Q4"
+            data_share_path: "2022Q4_20240426T113151Z"
```

Comment:
The path seems like it should live in the `config.yml`? to be in line with `workflow.data.preparation` and friends?

Reply:
See https://github.com/RMI-PACTA/workflow.data.preparation/blob/afdd9e08ee74a6b44d34869e57295acfa0430d84/config.yml#L181-L187

Reply:
This repo doesn't use a `config.yml` yet. I'm passing it as a default in the matrix here, since I'm expecting that the main way this workflow gets called is form `workflow.data.prep` (where it will get called from the build configs).

Reply:
Overall, this workflow running from this repo is mostly a smoke test to ensure that it works, rather than to produce actual results that are intended to be used.

Reply:
https://github.com/RMI-PACTA/workflow.prepare.pacta.indices/blob/main/config.yml ?

Reply:
I have some pretty seriously strong reservations against `workflow.data.preparation` being an appropriate place to call this workflow. 

It's critically important that this workflow is called from the appropriate PACTA docker image, but there is no benefit in requiring a run of data prep any time we want to run indices 

(imagine the case that `portfolio.pacta.allocate` gains commits. We would need to re-run indices, but there would be no need to re-run data prep)

Reply:
Hmm. good point. I still think that given the pacta-data set is highly variable (compared to the other params specified in the config) it's better to have it be a variable input, rather than specifying in the config.

To be entirely honest, reading out the values from the config.yml is something that would add a lot of complexity, since we would need to use `yq` rather than the built-in `jq` (if we had `json` configs), but possibly worth flagging as a future change.

Reply:
This workflow should be merged with `workflow.pacta` or `workflow.pacta.report` (once either of those repos is live), but I don't think `workflow.data.preparation` is the place to live. 

Anyway, feel free to merge this as-is if you wish, I think this whole repo needs a lot of re-thinking to begin with, and I don't want to block indices prep for GENERAL 22Q4

Reply:
> I have some pretty seriously strong reservations against `workflow.data.preparation` being an appropriate place to call this workflow.

you're absolutely correct. I wrote the wrong thing. I meant that this is intended for int egration with `workflow.transition.monitor` as part of the build pipeline there:

1. build the GHCR image
2. start index prep
3. use the fresh indices as part of the private (ACR) image build

this would keep the indices up to date with both the pacta code and the pacta-data used in the CTM image.

Reply:
Yes! 100% agree with that approach

Reply:
https://github.com/RMI-PACTA/workflow.prepare.pacta.indices/issues/95